### PR TITLE
Add support for Macedonian language

### PR DESCRIPTION
**Pull request by dimegalev**
There is a high chance that you don't know the Macedonian language but it's really not that far off other Slavic languages, please have a look at let me know if I need to make any changes.

Cheers

### CODE REVIEW

File: src/Concerns/ManagesNumberTransformers.php [hunk 1]
```diff
@@ -28,6 +28,7 @@ trait ManagesNumberTransformers
         'ka' => Transformer\GeorgianNumberTransformer::class,
         'lt' => Transformer\LithuanianNumberTransformer::class,
         'lv' => Transformer\LatvianNumberTransformer::class,
+        'mk' => Transformer\MacedonianNumberTransformer::class,
```
Comment (kwn) [inline src/Concerns/ManagesNumberTransformers.php:31]: Could you please update the README.md file as well?

File: src/Legacy/Numbers/Words/Locale/Mk.php [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+    }
+
+    private function initDigits()
+    {
+        if (!self::$digitsInitialized) {
+            for ($i = 3; $i <= 9; $i++) {
+                self::$digits[1][$i] =& self::$digits[0][$i];
+            }
+            for ($i = 3; $i <= 9; $i++) {
+                self::$digits[-1][$i] =& self::$digits[0][$i];
```
Comment (kwn) [inline src/Legacy/Numbers/Words/Locale/Mk.php:174]: Same as above.

**Review by kwn — CHANGES_REQUESTED** [review]
Thank you for the contribution. Could you please add some unit tests?

**Comment by dimegalev** [comment]
Thank you for your review, I'll have a look at the requested changes and do the changes as soon as I can.

**Comment by dimegalev** [comment]
@kwn I've made some changes, think I covered everything that was requested. Let me know if there are any other changes that you'd like to see done.

**Review by kwn — APPROVED** [review]
Thank you. It looks fantastic!

**Pull request merged**